specs/opcode/02MUL_04DIV_06MOD_1bSHL_1cSHR.md

```diff
@@ -9,10 +9,12 @@ Pop two EVM words `a` and `b` from the stack, and push `c` to the stack, where `
 - for opcode `MUL`, compute `c = (a * b) % 2^256`
 - for opcode `DIV`, compute `c = a // b` when `b != 0` otherwise `c = 0`
 - for opcode `MOD`, compute `c = a mod b` when `b != 0` otherwise `c = 0`
+- for opcode `SHL`, `b` is a number of bits to shift to the left, compute `c = (a * 2^b) % 2^256` when `b < 256` otherwise `c = 0`
+- for opcode `SHR`, `b` is a number of bits to shift to the right, compute `c = a // 2^b` when `b < 256` otherwise `c = 0`
 
 ### Circuit behavior
 
-To prove the `MUL/DIV/MOD` opcode, we first construct a `MulAddWordsGadget` that proves `a * b + c = d (mod 2^256)` where `a, b, c, d` are all 256-bit words.
+To prove the `MUL/DIV/MOD/SHL/SHR` opcode, we first construct a `MulAddWordsGadget` that proves `quotient * divisor + remainder = dividend (mod 2^256)` where `quotient, divisor, remainder, dividend` are all 256-bit words. Rename `quotient, divisor, remander, dividend` to `a, b, c, d` for simple as below.
```

Comment:
At the end of this line, I'm not sure what "simple" is referring to?

Reply:
Fixed the last sentence as:
```
Consider `quotient, divisor, remander, dividend` as `a, b, c, d` in `MulAddWordsGaget`.
```
It seems that the variable names could be same in `MulAddWordsGadget` code, and short as the below table in Markdown doc.